doc/source/visualizing/plots.rst

```diff
@@ -1054,6 +1055,68 @@ change the property of a single line, give also the index of the profile.
     # change only the first line
     plot.set_line_property("linestyle", "--", 0)
 
+.. _how-to-1d-unstructured-mesh:
+
+1D Line Sampling
+----------------
+
+YT has the ability to sample datasets along arbitrary lines
+and plot the result. You must supply five arguments to the ``LinePlot``
+class. They are enumerated below:
```

Comment:
Sure, [this](http://yt-project.org/docs/dev/reference/api/yt.data_objects.selection_data_containers.html#yt.data_objects.selection_data_containers.YTRay) is what I'm thinking of.